### PR TITLE
Extended acts section with location key

### PR DESCRIPTION
**Pull request by MathWay**
New key added with corresponding updates around 'acts' section parsing.

### CODE REVIEW

File: context.c [hunk 1]
```diff
@@ -133,8 +133,16 @@ struct ds_ctx* ds_ctx_load(const char* _config_file)
...
+		// location
+		const char* location = ds_cfg_try_get_cstr(cfg, sections[i], DS_CFG_KEY_LOC);
+		if (!location || pfcq_strlcmp(location, DS_CFG_LOC_INLINE) != 0)
+		{
+			inform("Action section: %s\n", ret->acts[c_x].name);
+			stop("Incorrect location specified");
+		}
+
 		char** keys = ds_cfg_get_keys(cfg, sections[i], (int*)&ret->acts[c_x].nact_items);
-		if (!--ret->acts[c_x].nact_items) // minus "type"
```
Comment (pfactum) [inline context.c]: This decrement must be preserved and converted into a separate decrement of -2. Its value is then used for malloc'ing (check the subsequent pfcq_alloc() call) and traversing, so for malloc'ing it must reflect the real amount of act items, and for traversing it must be +2'ed to cover all the keys.

File: context.c [hunk 2]
```diff
@@ -133,8 +133,16 @@ struct ds_ctx* ds_ctx_load(const char* _config_file)
 		// name
 		ret->acts[c_x].name = pfcq_strdup(sections[i]);
 
+		// location
```
Comment (pfactum) [inline context.c]: Consider adding some "TODO:" here regarding future addition of "mysql" value too.

Comment (MathWay) [reply context.c]: ok

**Comment by pfactum** [comment]
Closing this since the config subsystem is rewritten entirely.

**Pull request closed without merging**